zkevm/Cargo.toml

```diff
@@ -47,6 +47,7 @@ procfs = "0.13.0"
 default = []
 #default = ["prove_verify"]
 prove_verify = []
+tachyon = []
```

Comment:
Please remove this line since I think this commit should only contain bump up part.